setup.py

```diff
@@ -35,7 +35,6 @@
     "Intended Audience :: Science/Research",
     "Development Status :: 5 - Production/Stable",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 3.5",
```

Comment:
Make sure to change the following line also:

https://github.com/pydicom/pydicom/blob/ba939ef2007c91d2b0dc741fdb0d3e81f066b3c5/setup.py#L81

Reply:
> Make sure to change the following line also:
> 
> https://github.com/pydicom/pydicom/blob/ba939ef2007c91d2b0dc741fdb0d3e81f066b3c5/setup.py#L81

Right - I knew that, they it got lost in doing something else...  Will do.